editor/create_dialog.h

```diff
@@ -102,7 +103,7 @@ class CreateDialog : public ConfirmationDialog {
 	void set_preferred_search_result_type(const String &p_preferred_type);
 	String get_preferred_search_result_type();
 
-	void popup_create(bool p_dont_clear, bool p_replace_mode = false);
+	void popup_create(bool p_dont_clear, bool p_replace_mode = false, StringName *original_node_class = NULL);
```

Comment:
Most of the code uses `const StringName&` :smiley:

Reply:
On it right now!